packages/wonder-blocks-timing/src/util/__tests__/timeout.test.js

```diff
@@ -110,12 +116,15 @@ describe("Timeout", () => {
 
         it("should invoke setTimeout to call the given action", () => {
             // Arrange
+            const timeoutSpy = jest.spyOn(global, "setTimeout");
             const action = jest.fn();
-            const timeout = new Timeout(() => action(), 500);
+            const timeout = new Timeout(
+                () => action(),
+                500,
+                SchedulePolicy.OnDemand,
+            );
```

Comment:
Why the switch to `OnDemand` for this test?

Reply:
Because that was what the test was written to work with, but it didn't use it and so it fails.